<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Name to changel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leditSourseVar"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>new naml</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leditResultVar"/>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRename">
       <property name="text">
        <string>Rename</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="formatCodeButton">
       <property name="text">
        <string>CodeFormat</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextEdit" name="teditSourceCode"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
